Dockerfile

```
@@ -965,11 +965,11 @@ FROM base-runner AS trustlab-runner
 
 # PNPM
 # symlink some dependencies
-COPY --from=trustlab-builder --link --chown=nextjs:nodejs /workspace/node_modules ./node_modules
+COPY --from=trustlab-builder --chown=nextjs:nodejs /workspace/node_modules ./node_modules
 
 # Next.js
 # Public assets
-COPY --from=trustlab-builder --chown=nextjs:nodejs /workspace/apps/trustlab/publi[c] ./apps/trustlab/public
+COPY --from=trustlab-builder --chown=nextjs:nodejs /workspace/apps/trustlab/public ./apps/trustlab/public
```

```suggestion
COPY --from=trustlab-builder --chown=nextjs:nodejs /workspace/apps/trustlab/publi[c] ./apps/trustlab/public
```

The reason for `publi[c]` is so that if the folder public doesn't exist, the `COPY` instruction should not throw an error.